src/components/FullPlayground.tsx

```diff
@@ -378,7 +375,7 @@ function ApolloedPlayground(props: { withRouter?: any }) {
   const datastore = usePlaygroundDatastore();
   const Router = props.withRouter ? props.withRouter : Box;
   return (
-    <Router basename={process.env.PUBLIC_URL}>
```

Comment:
`process.env` isn't a thing in Vite in the same way it was in webpack. I went through and modified each instance. In the `PUBLIC_URL` case it wasn't necessary because root-relative is basically always fine.